### PR TITLE
[Docs] Rename remaining Swift Flow refs. Announce merger.

### PR DESCRIPTION
**Pull request by agentk**

### CODE REVIEW

**Comment by agentk** [comment]
I’m not sure why the travis if failing. It’s only docs changes.

**Comment by agentk** [comment]
Oh. It's just a travis timeout. Should be fixed once it reruns.

File: README.md [hunk 1]
```diff
@@ -106,11 +111,11 @@ The `newState` method will be called by the `Store` whenever a new app state is
 
 Button taps result in dispatched actions that will be handled by the store and its reducers, resulting in a new app state.
 
-This is a very basic example that only shows a subset of Swift Flow's features, read the Getting Started Guide to see how you can build entire apps with this architecture.
+This is a very basic example that only shows a subset of ReSwift's features, read the Getting Started Guide to see how you can build entire apps with this architecture.
 
-[You can also watch this talk on the motivation behind Swift Flow](https://realm.io/news/benji-encz-unidirectional-data-flow-swift/).
+[You can also watch this talk on the motivation behind ReSwift](https://realm.io/news/benji-encz-unidirectional-data-flow-swift/).
```
Comment (dcvz) [inline README.md]: maybe make a note that it was Swift-Flow before hand? just for clarity for users not familiar with the framework already.

File: ReSwift.podspec [hunk 2]
```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   s.version          = "0.2.3"
   s.summary          = "Unidirectional Data Flow in Swift"
   s.description      = <<-DESC
-                        Swift Flow is a Redux-like implementation of the unidirectional data flow architecture in Swift.
+                        ReSwift is a Redux-like implementation of the unidirectional data flow architecture in Swift.
```
Comment (dcvz) [inline ReSwift.podspec]: Should this maybe match the description on Github?

Comment (agentk) [inline ReSwift.podspec]: Good catch. I'll make a note of it for the next pod release.

**Comment by Ben-G** [comment]
@agentk Will need a rebase before we can merge :smile:

**Comment by agentk** [comment]
Rebase done. I haven't accounted for SwiftFlow/CoreTypes/StoreType.swift from the rebase. That might be best for a artefact cleanup PR.

**Comment by Ben-G** [comment]
Merci! :ship:

**Pull request merged**